lib/model.js

```diff
@@ -2573,7 +2581,7 @@ class Model {
         }
       }
 
-      if (options.ignoreDuplicates && ['mssql'].includes(dialect)) {
+      if (options.ignoreDuplicates && ['mssql', 'db2'].includes(dialect)) {
```

Comment:
I wonder if we should move this to the dialect's supports area and use `dialect.supports.ignoreDuplucates` here?

Reply:
@sdepold can be done but need to update mssql specific code too accordingly. We just followed the mssql code and added 'db2'.

Reply:
ok. leave it as is for now and I'll have a look when I factor the things out of the code base